elastic/kibana
# [Fleet] Link to the installed version of an integration from global search

## Summary

Closes #114607 
We previously linked to the latest version of an integration from global search, removing the version from the link means we will now navigate to the installed version if there is one, else the latest version.

To test:

- install an earlier version of a package
- use the global search bar to search for that package name
- you should be linked to the installed version of the package not the latest version
